src/util.js

```diff
@@ -74,18 +74,25 @@ util.getAttrs = function( tagMarkup, settings )
     }
 };
 
-util.getRemote = function( uri, callback, toDataUri )
+function getRemote( uri, settings, callback, toDataUri )
 {
     if( /^\/\//.test( uri ) )
     {
         uri = "https:" + uri;
     }
 
+    var requestOptions = {
+        uri: uri,
+        encoding: toDataUri ? "binary" : ""
+    };
+
+    if( typeof settings.requestTransform === "function" )
+    {
+        requestOptions = settings.requestTransform( requestOptions ) || requestOptions;
```

Comment:
The `|| requestOptions` seems a little odd here. If for some reason someone writes a `requestTransform` function that returns a falsey object, why should that then default to `requestOptions` as defined above? I feel that should either be an error or indicate not to make the request at all. Thoughts?